docker/Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN yum update -y && yum install -y centos-release-scl epel-release && \
     tar xzf zookeeper.tar.gz -C /deps/src && \
     rm -v ./*.tar.gz && \
     /setup_deps.sh -a "$TARGETARCH" -z "$ZETASQL_VERSION" -t "$THIRDPARTY_VERSION" && \
-    rm -v /setup_deps.sh
+    rm -v /setup_deps.sh && \
+    yum install -y rsync
```

Comment:
I prefer the compile image as minimal, rsync is not required dep based on current workflow

Reply:
maybe we can keep rsync in Dockerfile_offline and remove it in Dockerfile?

Reply:
It is okay to delete this since we need to connect Internet to deploy.